src/components/Import.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react'
+
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import Modal, {
+  ModalHeader,
+  ModalDescription,
+  ModalFooter
+} from 'cozy-ui/transpiled/react/Modal'
+import { Button } from 'cozy-ui/transpiled/react/Button'
+import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
+import TextField from 'cozy-ui/transpiled/react/MuiCozyTheme/TextField'
+
+import viewStyles from 'styles/view'
+import formStyles from 'styles/fields'
+import ReactMarkdownWrapper from 'components/ReactMarkdownWrapper'
+
+const importImage = require('assets/images/import-cozy.svg')
+
+class Import extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      displayModal: false,
+      url: ''
+    }
+    this.toggleModal = this.toggleModal.bind(this)
+    this.setURL = this.setURL.bind(this)
+    this.submitImport = this.submitImport.bind(this)
+  }
+
+  toggleModal() {
+    this.setState(currentState => ({
+      displayModal: !currentState.displayModal
+    }))
+  }
+
+  setURL(event) {
+    this.setState({ url: event.target.value })
+  }
+
+  submitImport() {
+    this.props.precheckImport(this.state.url)
+  }
+
+  render() {
+    const { t, importData } = this.props
+    const { displayModal } = this.state
+    return (
+      <div>
+        <div className={viewStyles['set-view-section']}>
+          <p className="u-mt-1">
+            <Button
+              onClick={this.toggleModal}
+              label={t('ProfileView.import.link')}
+              extension="full"
+            />
+          </p>
+        </div>
+        {displayModal && (
+          <Modal
+            className={viewStyles['set-export-modal']}
+            secondaryAction={this.toggleModal}
+            mobileFullscreen
+          >
+            <ModalHeader className="u-ta-center u-pr-0">
+              <img
+                className={viewStyles['set-export-modal-image']}
+                alt={t('ProfileView.import.modal.title')}
+                src={importImage}
+              />
+              <h2 className={viewStyles['set-export-modal-title']}>
+                {t('ProfileView.import.modal.title')}
+              </h2>
+            </ModalHeader>
+            <ModalDescription
+              className={viewStyles['set-export-modal-description']}
+            >
+              <ReactMarkdownWrapper
+                source={t('ProfileView.import.modal.description')}
+              />
+              <MuiCozyTheme>
```

Comment:
You should not add it there. It should be added at the top of the React Tree. I think https://github.com/cozy/cozy-settings/blob/master/src/targets/browser/index.jsx#L56 is a good place 👍

Reply:
Please add it also for the intents, so that browser/index.jsx and intsn/index.jsx stay mostly the same.